### ISSUE TITLE
cli issues on Windows 10

### ISSUE THREAD

**Issue by viktorlarsson** [vudash/vudash#61]
Hi!

Just tried it out on Windows, but i'm having some issues with vudash create. 

You are using template literals, but I haven't seen ``` before (except for Markdown on Github) in cli/src/create.js

https://github.com/vudash/vudash/blob/448e56732a920e6e9e8e792eaf074934473ae157/packages/core/src/cli/create.js#L10-L23

```const dockerFileContents = ```
                             ^
...
TypeError: "" is not a function
    at Object.<anonymous> (C:\Projects\dashboard-test\node_modules\vudash\src\cli\create.js:10:30)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\l19550635\Documents\Projects\lindex-dashboard\node_modules\vudash\bin\vudash.js:3:16)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)```

When use single literals it seems to start working. Is there any special reason for this?

```
const dockerFileContents = `
FROM node:9-alpine

COPY . /app

WORKDIR /app
RUN npm install

EXPOSE 3300

ENV SERVER_URL http://localhost:3300

CMD npm start
`
```

**Comment by antony** [comment]
Hi @viktorlarsson,

This is a very good point. It should be using single quotes. (Groovy on the JDK also uses triples!)

Will fix and release a patch update ASAP.

**Comment by antony** [comment]
Fixed in `9.8.3`. Apologies - this was a very fast and loose change, and I hadn't properly tested it!

**Comment by viktorlarsson** [comment]
No worries, glad I could help!
